packages/fdc3-agent-proxy/test/features/private-channels-deprecated.feature

```diff
@@ -37,7 +37,7 @@ Feature: Basic Private Channels Support
     And I call "{theListener}" with "unsubscribe"
     Then messaging will have posts
       | type                                          | payload.listenerType | payload.privateChannelId | payload.listenerUUID | matches_type                                  |
-      | privateChannelAddEventListenerRequest         | onUnsubscribe        | {privateChannel.id}      | {null}               | privateChannelAddEventListenerRequest         |
```

Comment:
I don't think so: 1) Checking the output wouldn't affect the test coverage on the test result, 2) this test covers the addition of a listener for the unsubscribe event (where the event type name changed recently in #1459), rather than calling `aContextListener.unsubscribe()` or `anEventListner.unsubscribe()`.

Nevertheless, I'll have a look for the untested call and rectify...

Turns out EventListener is *only* used for FDC3 API events, which only includes the channel changed event currently - adjusting the name (to ApiEventListener? is that any better?) may help with confusion there. You use EventListener to listen for channel changes to apply in DefaultChannelSupport (where it's registered but never removed). Its also used for app channel changed event listeners - so we're missing a test for that. Added one.